src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1209,7 +1209,7 @@ private void updateOverallHappiness()
             final BuildingHome home = citizen.getHomeBuilding();
             if(home != null)
             {
-                housing = home.getBuildingLevel();
+                housing += home.getBuildingLevel();
```

Comment:
The level number for citizen huts is the number of citizen the hut can hold.
In other word, If your colony has more room for citizen than citizen, this will increase the happiness.
@Raycoms Could you comment on this.
I would have assumed that most of the time the number of citizen = the number of "beds" anyway so we would be at 1 most of the time.
Also why does it not decrease happiness when we don't have enough home ?